include/z64audio.h

```diff
@@ -125,7 +125,7 @@ typedef struct {
     /* 0x00 */ u32 medium : 2;
     /* 0x00 */ u32 unk_bit26 : 1;
     /* 0x00 */ u32 unk_bit25 : 1;
-    /* 0x01 */ u32 size : 24;
+    /* 0x01 */ size_t size : 24;
```

Comment:
this change appears to be an error, as size is set to 24 bits, changing from u32 to size_t in 64 bit mode would not make sense.

Reply:
its still a size so it should stay size_t imo